integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -290,7 +290,7 @@ case class CarbonLoadDataCommand(
           FileFactory.mkdirs(metadataDirectoryPath)
         }
       } else {
-        carbonLoadModel.setSegmentId(System.currentTimeMillis().toString)
+        carbonLoadModel.setSegmentId(System.nanoTime().toString)
```

Comment:
Many times have observed (mainly from user environment) that , two concurrent tasks can get same nano time. So, we use UUID in these scenario. But for segmentID we cannot use as it as may be  segment id will be casted to long. 

Need other solutions or go back to old look up logic

Reply:
Added lookup for non-transactional also..